apps/web/app/(basenames)/name/[username]/renew/page.tsx

```diff
@@ -12,6 +13,10 @@ export default async function Page(props: PageProps) {
   const name = params.username.split('.')[0];
   const formattedName = await formatDefaultUsername(name);
 
+  if (isBasenameRenewalsKilled) {
+    return notFound();
```

Comment:
Another option could be to redirect to /names, but it feels more correct to just act like the route doesn't exist